services-js/access-boston/src/server/ping-templates/mockData.ts

```diff
@@ -0,0 +1,76 @@
+export default {
+  assetPath: 'assets/ping/assets',
+  name: 'name',
+  username: 'city-employee',
+  pass: 'pass',
+  newPass1: 'newPass2',
+  newPass2: 'newPass1',
+  PingFedBaseURL: '/',
+  forgotPasswordUrl: '/',
+  changePasswordUrl: '/',
+  enableRememberUsername: false,
+  rememberUsernameCookieExists: false,
+  enableCheckboxByDefault: false,
+  rememberUsernameChecked: 'checked',
+  usernameEditable: false,
+  supportsPasswordChange: false,
+  supportsPasswordReset: false,
+  registrationEnabled: false,
+  templateMessages: {
+    strings: {
+      title: 'Page Title',
+      usernameTitle: 'Username',
+      passwordTitle: 'Password',
+      newPassword1Title: 'New Password',
+      newPassword2Title: 'Confirm Password',
+      missingField: 'Please fill out this field',
+      rememberUsernameTitle: 'Remember Username',
+      signInButtonTitle: 'Sign In',
+      changeButtonTitle: 'Change Password',
+      cancelButtonTitle: 'Cancel',
+      changePassword: 'Change Password',
+      forgotPassword: 'Forgot Password',
+      noAccountMessage: 'No Account',
+      registerAccount: 'Register Account',
+      loginWithButtonTitle: 'Login',
+      info: 'You have now been logged out.',
+      headerMessage: 'Change Password',
+    },
+    // @ts-ignore
+    getMessage: function(prefix, message) {
+      return this.strings[message];
+    },
+  },
+  locale: {
+    getLanguage: () => 'en',
+  },
+  orientation: 'ltr',
+  url: '',
+  authnMessageKey: '',
+  errorMessageKey: '',
+  serverError: '',
+  altAuthSources: {
+    authSource: [],
+    size: () => 0,
+  },
+  loginFailed: false,
+  adapterIdField: 'adapterId Field',
+  adapterId: 'adapterId',
+  localizedMessageResolver: {
+    strings: {
+      locale: 'en',
+    },
+    // @ts-ignore
```

Comment:
why do you need these? I'd rather you drop in `any` types or something rather than turn off type checking altogether.

Reply:
The functions being mocked expect two parameters, and I wanted to keep the mocks as similar as possible. Since we’re not using the first parameter in these cases, TS throws a `[parameter] is declared but its value is never read` error. I wasn‘t sure what would be a cleaner solution (but am open to suggestions!).

Reply:
If you prefix the parameter name with `_` then TS won't give usage complaints.

Reply:
Oh wow, TIL. Thanks!